test/releasers/utils.ts

```diff
@@ -34,3 +35,31 @@ export function buildGitHubFileRaw(content: string): GitHubFileContents {
     sha: crypto.createHash('md5').update(content).digest('hex'),
   };
 }
+
+/**
+ * Stub files on github with fixtures.
+ *
+ * @param fixturePath - Parent dir of fixture files. Fixture files must be
+ *  direct children (no sub folders).
+ * @param sandbox - created in test file.
+ * @param gh - GitHub instance's `getFileContentsOnBranch` to stub
+ * @param files - list of file paths from repo root to stub
+ * @param defaultBranch - branch arg to `getFileContentsOnBranch`
+ */
+export function stubFilesFromFixtures(
+  fixturePath: string,
+  sandbox: SinonSandbox,
+  gh: GitHub,
+  files: string[],
+  defaultBranch = 'master'
+) {
+  const stub = sandbox.stub(gh, 'getFileContentsOnBranch');
```

Comment:
I like this abstraction, if I'm following properly, it will always use the filename and drops the leading path in cases like `packages/foo/package.json`?

Reply:
yeah, I should clarify the `@param files` argument more clearly as it's dual purpose. If you pass
```javascript
['packages/foo/package.json']
```
(the path in the fake repo under test and used in `stub.withArgs`) then the assumption is that there's a file in the real `googleapis/release-please` named
```javascript
`${fixturePath}/package.json`
```